lightweightsemaphore.h

```diff
@@ -253,6 +255,80 @@ class Semaphore
 		}
 	}
 };
+#elif defined(__MVS__)
```

Comment:
Why can't the Linux version be used for MVS as well? Looks like both are POSIX based.

Reply:
To be honest, I used a new block in case there are platform-specific quirks to handle.
In this case, chrono bits from unix aren't used.
Perhaps I can just exclude that with a check..
Could this be accepted for now, I'll be sure to revisit and send another PR when I can work it out cleanly.

Reply:
What chrono bits? The implementation is the same except `MOODYCAMEL_LIGHTWEIGHTSEMAPHORE_MONOTONIC` will be undefined. Unless I'm missing something.

Reply:
Oh wow, I didn't look at the whole file at all.
It seems I've just sent this out based on the third party "version" of this imported in DuckDB - https://github.com/duckdb/duckdb/blob/master/third_party/concurrentqueue/lightweightsemaphore.h

... and that is 2 years old.

Will push updates soon..

Reply:
Thanks for taking time to review & guide.
Does it look ok for merge now?